app/jobs/post_grouper.rb

```diff
@@ -0,0 +1,19 @@
+#class PostGrouper < ActiveJob::Base
+class PostGrouper
+  def perform(cleanup_gene_claims = true)
```

Comment:
Fixed. The delete_orphaned_gene_claims will only delete gene claims from Ensembl so that's an additional safeguard. In your opinion, is the `should_cleanup_gene_claims?` `true` in `app/jobs/update_entrez.rb` correct?